src/std/misc/func.ss

```diff
@@ -182,6 +182,13 @@
   ((recur f ... fn)
    (call-with-values (lambda () (recur f ...)) fn)))
 
+;; pred-not returns #f if pred returns #t, and #t if pred returns #f.
+;;
+;; Example:
+;; (filter (pred-not number?) '(a 1 b 2 c)) => (a b c)
+(def (pred-not pred)
+  (lambda (v) (not (pred v))))
```

Comment:
this is simply `(? (not pred))`, we already have shorthand notation.

Reply:
Oh, fine :)